torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -178,14 +180,24 @@ def _create_op_overload_to_exporter_key_table() -> (
                 # different exporter keys.
 
                 table[op_overload] = op_overload_packet._qualified_op_name
-    # TODO(justinchuby): is baddbmm different?
-    table[torch.ops.aten.baddbmm.default] = "aten::baddbmm"
+    # NOTE: Below are not in torch.ops.aten/torch.ops.prim
+    # TODO(titaiwang): aten::sym_size has overload, but fx graph is using
+    # overloadpacket for some reasons.
+    # https://github.com/pytorch/pytorch/issues/97201
+    table[torch.ops.aten.sym_size.int] = "aten::sym_size"
```

Comment:
We went over this offline. Did you find that this is still required? i.e. `# NOTE: Below are not in torch.ops.aten/torch.ops.prim` this is true for `torch.ops.aten.sym_size.int`?

To be precise, the below 3 lines of comment is unrelated to this particular line, and should be put in `fx_to_onnxscript.py` (which you already did).
```
    # TODO(titaiwang): aten::sym_size has overload, but fx graph is using
    # overloadpacket for some reasons.
    # https://github.com/pytorch/pytorch/issues/97201
```

Reply:
Done